Sources/CardPayments/Models/CardResult.swift

```diff
@@ -9,8 +9,10 @@ public struct CardResult {
     /// The order ID associated with the transaction
     public let orderID: String
 
-    // TODO: parse contents of this URL once we are clear on values returned from 3ds
     /// This is the deep link url returned from 3DS authentication
     @_documentation(visibility: private)
     public let deepLinkURL: URL?
+    
+    /// Liability shift value returned from 3DS verification
+    public let liabilityShift: String?
 }
```

Comment:
I'm thinking about the results we return for card vaulting also. For non-3ds contingency flow, we have the "APPROVED" status from confirm-payment-source. Should we return "APPROVED" status for that flow and liabilityShift for 3DS flow?

Reply:
So CardResult/CardVaultResult would have optional liabilityShift and optional status field.
What are your thoughts?

Reply:
We used to have "status" field for CardResult but we removed it since we don't have access now to status after 3DS flow in SDK now.

Reply:
If we remove the `status` field from `CardResult` for some reason, I believe it's not necessary to add it back right now. If the Vaulting flow with 3DS requires it, we can reintroduce it later

Reply:
I think it's a bit weird to just return orderID when we know we have "approved" status for non 3DS flow. We had removed it because we didn't have reqs on 3DS clear at the time.

Reply:
We can align with Android and just keep what we are returning and discuss maybe adding it later. I'm good with that.